docs/reuse/tutorial/setup_edge.rst

```diff
@@ -18,31 +18,46 @@ Once the VM is up, open a shell into it:
 
     multipass shell charm-dev
 
+Unless stated otherwise, we will work entirely within the VM from now on.
+
+Install Rockcraft and Charmcraft
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. admonition:: This extension is under active development
+
+    To follow this tutorial, you must install ``latest/edge``
+    of Rockcraft and Charmcraft.
```

Comment:
Since we describe installation specifics later, I think we can cover the why a bit more here:

```suggestion
    This tutorial requires the edge releases of Rockcraft and Charmcraft, which are 
    riskier than the stable releases.
```